shared/src/ledger/mod.rs

```diff
@@ -5,5 +5,6 @@ pub mod ibc;
 pub mod native_vp;
 pub mod parameters;
 pub mod pos;
+pub mod protocol_vps;
```

Comment:
can we standardize on "protocol vp" or "protocol vps" or something else, it's "mod protocol_vps" and "struct ProtocolVp" right now